beluga/cmake/Config.cmake.in

```diff
@@ -1,11 +1,14 @@
 @PACKAGE_INIT@
 
+list(APPEND CMAKE_MODULE_PATH "@OPENVDB_CMAKE_MODULE_PATH@")
+
 include(CMakeFindDependencyMacro)
 find_dependency(Eigen3 REQUIRED NO_MODULE)
 find_dependency(range-v3 REQUIRED)
 find_dependency(HDF5 COMPONENTS CXX REQUIRED)
 find_dependency(Sophus REQUIRED)
 find_dependency(TBB REQUIRED)
+find_dependency(OpenVDB QUIET)
```

Comment:
I changed in a61feed6a6f6886dca81cd62efcd3ee2e2a8f7e8.

`if("@USE_OPENVDB@")
    list(APPEND CMAKE_MODULE_PATH "@OPENVDB_CMAKE_MODULE_PATH@")
    find_dependency(OpenVDB REQUIRED)
endif()`

But i think it is not relevant now, since it will moved to another package.